bin/whoami

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/perl
+
+=encoding utf8
+
+=begin metadata
+
+Name: whoami
+Description: display effective user ID
+Author: Oğuz Ersen, oguzersen@protonmail.com
+License: perl
+
+=end metadata
+
+=cut
+
+use strict;
+
+my $user = getpwuid($>);
+print "$user\n";
+
+=head1 NAME
+
+whoami - display effective user ID
+
+=head1 SYNOPSIS
+
+B<whoami>
+
+=head1 DESCRIPTION
+
+Print the username associated with the current effective user ID,
+same as B<id -un>.
+
+=head1 AUTHOR
+
+Oğuz Ersen (oguzersen@protonmail.com)
+
+=head1 SEE ALSO
+
+whoami(1), id(1)
+
+=head1 COPYRIGHT and LICENSE
+
+Copyright (c) by Oğuz Ersen 2021.
+
+This program is free and open software. You may use, modify, distribute,
+and sell this program (and any modified variants) in any way you wish,
```

Comment:
Note that the declared license in the metadata is "perl", so there are some restrictions there.

Reply:
I have copied and pasted this from other commands, e.g. `bin/awk` has the same. Is it OK if I change this section with (also copied and pasted from other commands, i.e. `bin/fold`):

    This code is freely modifiable and freely redistributable under Perl's
    Artistic License.

Reply:
It would be better to simply say "This code is licensed under the Artistic License 2."